machine_management/manually-scaling-machineset.adoc

```diff
@@ -15,7 +15,7 @@ see xref:../machine_management/modifying-machineset.adoc#modifying-machineset[Mo
 
 .Prerequisites
 
-* If you enabled the cluster-wide proxy and scale up workers not included in `networking.machineCIDR` from the installation configuration, you must xref:../networking/enable-cluster-wide-proxy.adoc#nw-proxy-configure-object_config-cluster-wide-proxy[add the workers to the Proxy object's `noProxy` field] to prevent connection issues.
+* If you enabled the cluster-wide proxy and scale up workers not included in `networking.machineNetwork.cidr` from the installation configuration, you must xref:../networking/enable-cluster-wide-proxy.adoc#nw-proxy-configure-object_config-cluster-wide-proxy[add the workers to the Proxy object's `noProxy` field] to prevent connection issues.
```

Comment:
Strictly to speak, I do not think `networking.machineCIDR` is replaced by `networking.machineNetwork.cidr`, but `networking.machineNetwork`, and the key accepts an array, items in this array is key=value pair.

Reply:
Just like the following example shows, `networking.machineNetwork[0].cidr` should be more accurate.

Reply:
proxy? I think it care about the whole array list from networking.machineNetwork's value

Reply:
yes, all the values of cidr in networking.machineNetwork would be added into `noProxy`

Reply:
Thanks!